src/components/SelectionList/InviteMemberListItem.tsx

```diff
@@ -0,0 +1,134 @@
+import Str from 'expensify-common/lib/str';
```

Comment:
Should we update UserListItem to re-use in this case instead of creating a new Component? Because most of the logic is the same

Reply:
@shubham1206agra As I understand, the point of this PR is to move the specific logic out of BaseListItem as mentioned [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1710332643953949?thread_ts=1710332452.771549&cid=C01GTK53T8Q). In here, I think we should combine into 1 component to avoid duplicated code

cc @burczu @mountiny